primes.rb

```diff
@@ -0,0 +1,12 @@
+def is_prime?(num)
+	start = num / 2
+	start.downto(2) do |n|
```

Comment:
One fun little trick is that you only need to test numbers from 2 up to the square root of the given number. That would save a few loops on big numbers.

Looks good, though!